test/stdlib/TimeExtension_test.rb

```diff
@@ -0,0 +1,119 @@
+require_relative "test_helper"
```

Comment:
[note] Compared with `Time_test.rb`, the file name `TimeExtension_test.rb` may be confusing... 🤔 
I'm open for a better name suggestion.